doc/build/orm/cascades.rst

```diff
@@ -95,7 +95,7 @@ becomes part of the state of that :class:`.Session`::
 
 ``save-update`` has the possibly surprising behavior which is that
 persistent objects which were *removed* from a collection
```

Comment:
agree sentence is bad, here is proposed wording:


`save-update`` has the possibly surprising behavior which is that persistent objects which were either *removed* from a collection, or de-associated from a scalar attribute, may nonetheless be pulled into the :class:`_orm.Session` of the parent object from which it's being removed; this is so that...

hows that?

Reply:
It's definitely better, but I think it's a little long. There's a lot to say and it's hard to do it all in one sentence. I tried breaking it up a little:

A ``save-update`` cascade can exhibit surprising behavior when removing an item from a collection or de-associating an object from a scalar attribute. In some cases, the orphaned objects may still be pulled into the ex-parent's session; this is so that...

Reply:
great, send that to press please

Reply:
I updated the patch-1 branch to reflect the new changes. Should be good to go